src/popup/pages/TipsClaim.vue

```diff
@@ -70,83 +71,88 @@ export default {
   },
   setup(props, { root }) {
     const { activeAccount } = useAccounts({ store: root.$store });
+    const { openModal, openDefaultModal } = useModals();
+    const { getTippingContracts } = useTippingContracts({ store: root.$store });
 
-    return {
-      activeAccount,
-    };
-  },
-  data: () => ({
-    url: '',
-    loading: false,
-    BLOG_CLAIM_TIP_URL,
-  }),
-  computed: {
-    ...mapState(['tippingV1']),
-    ...mapGetters('sdkPlugin', ['sdk']),
-    ...mapGetters(['tippingSupported']),
-    normalizedUrl() {
-      if (!validateTipUrl(this.url)) return '';
-      return toURL(this.url).toString();
-    },
-  },
-  async mounted() {
-    if (IS_EXTENSION) {
-      const [tab] = await browser.tabs.query({ active: true, currentWindow: true });
-      if (tab && validateTipUrl(tab.url)) {
-        this.url = tab.url;
-      }
-    }
-  },
-  methods: {
-    async claimTips() {
-      const { openModal, openDefaultModal } = useModals();
-      const { activeAccount } = useAccounts({ store: this.$store });
-
-      const url = this.normalizedUrl;
-      this.loading = true;
-      await watchUntilTruthy(() => this.sdk && this.tippingV1);
+    const tipUrl = ref('');
+    const loading = ref(false);
+    const tippingSupported = computed(() => root.$store.getters.tippingSupported);
+
+    const normalizedUrl = computed(() => (
+      validateTipUrl(tipUrl.value)) ? toURL(tipUrl.value).toString() : '');
```

Comment:
It's more common for multiline computed properties to split the lines in a following way:
```suggestion
    const normalizedUrl = computed(
      () => validateTipUrl(tipUrl.value) ? toURL(tipUrl.value).toString() : '',
    );
```